dev-python/elasticsearch-curator/elasticsearch-curator-5.8.1-r3.ebuild

```diff
@@ -0,0 +1,165 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_7 )
```

Comment:
Does it not support python3_8 and python3_9?
https://archives.gentoo.org/gentoo-dev/message/ba2fc4854ed6bb49813819ae6f5e9552

Reply:
It probably also compiles with python3_8 and python3_9. I just copied the existing -r2 ebuild and added the missing dependencies. 

Now I tried to change this line in a local overlay to `PYTHON_COMPAT=( python3_{7..9} )`
But then emerge demands a use flag change:

`# required by dev-python/elasticsearch-curator (argument)`
`>=dev-python/elasticsearch-curator-5.8.1-r3 -python_targets_python3_8`

I don't know why. All dependencies listed in the ebuild support at least python3_8.

Reply:
Looks like curator still has some problems with python3_8 (failing tests)
https://github.com/elastic/curator/pull/1512

Reply:
> Looks like curator still has some problems with python3_8 (failing tests)
> [elastic/curator#1512](https://github.com/elastic/curator/pull/1512)

Please see https://github.com/elastic/curator/issues/1587#issuecomment-764883050
You also want to bump to most recent version(s) which is presently [5.8.3](https://github.com/elastic/curator/releases/tag/V5.8.3) instead of sticking to 5.8.1

Reply:
> It probably also compiles with python3_8 and python3_9. I just copied the existing -r2 ebuild and added the missing dependencies.
> 
> Now I tried to change this line in a local overlay to `PYTHON_COMPAT=( python3_{7..9} )`
> But then emerge demands a use flag change:
> 
> `# required by dev-python/elasticsearch-curator (argument)`
> `>=dev-python/elasticsearch-curator-5.8.1-r3 -python_targets_python3_8`
> 
> I don't know why. All dependencies listed in the ebuild support at least python3_8.

You want use elasticsearch-curator-5.8.3 and add python3_9 to importlib_metadata-3.4.0 and elasticsearch-py-7.7.0

Reply:
I also had to add python3_9 to dev-python/click

I added a 5.8.3 ebuild for elasticsearch-curator, it compiled using python targets 3_7, 3_8 and 3_9.  Tested  it against my elasticsearch instance. Looks fine to me.

However when I run `repoman ci` on the importlib_metadata-3.4.0 change (adding python3_9), I get an error:

```
  dependency.bad [fatal]        80
  dev-python/importlib_metadata/importlib_metadata-3.4.0.ebuild: BDEPEND: ~alpha(default/linux/alpha/17.0)
[     '>=dev-python/importlib_resources-1.3.0[python_targets_pypy3(-)?,python_targets_python3_7(-)?,python_targets_python3_8(-)?,python_targets_python3_9(-)?,-python_single_target_pypy3(-),-python_single_target_python3_7(-),-python_single_target_python3_8(-),-python_single_target_python3_9(-)]']
   dev-python/importlib_metadata/importlib_metadata-3.4.0.ebuild: BDEPEND: ~alpha(default/linux/alpha/17.0/desktop)
[     '>=dev-python/importlib_resources-1.3.0[python_targets_pypy3(-)?,python_targets_python3_7(-)?,python_targets_python3_8(-)?,python_targets_python3_9(-)
...
```
I thought of adding python3_9 to importlib_resources, too. But this comment in the ebuild kept me from doing so:
>  This is a backport of Python 3.9's importlib.

I'm not a Python developer and don't want to mess with dependencies and version conflicts which I don't understand

Reply:
Anyway, I added the python3_9 dependency to importlib_resources and importlib_metadata to get past the CI analysis.

Please have a look at this change if its sensible.